localisation/00_wc_sad.csv

```diff
@@ -473,13 +473,13 @@ ancient_war;Ancient of War;;Urtum des Krieges;;;;;;;;;;;x
+creature_protector_desc;Ancients are giant, sentient demigod-like trees that serve as protectors of nature. They were among the first creatures of Azeroth, the first guardians of its life.;;Urtume sind gigantische, empfindsame halbgott�hnliche B�ume, die als Besch�tzer der Natur dienen. Sie geh�rten zu den ersten Gesch�pfen Azeroths, den ersten H�tern seines Lebens.;;;;;;;;;;;x
 creature_ancient_lore;Ancient of Lore;;Urtum der Weisheit;;;;;;;;;;;x
-creature_ancient_lore_desc;A type of ancient, the ancients of lore are the keepers of wisdom, and the living keys necessary to unlock nature's most guarded secrets.;;Die Urt�me der Weisheit sind die Bewahrer derselben und die lebendigen Schl�ssel zur Enth�llung der am besten gesch�tzten Geheimnisse der Natur.;;;;;;;;;;;x
+creature_ancient_lore_desc;A type of ancient, the ancients of lore are the keepers of wisdom, and the living keys necessary to unlock nature's most guarded secrets.;;Die Urtume der Weisheit sind die Bewahrer derselben und die lebendigen Schl�ssel zur Enth�llung der am besten gesch�tzten Geheimnisse der Natur.;;;;;;;;;;;x
 creature_ancient_war;Ancient of War;;Urtum des Krieges;;;;;;;;;;;x
-creature_ancient_war_desc;One of the toughest types of ancient, Ancients of War train night elves in the arts of weaponry.;;Als eine der z�hesten Arten der Urt�me bilden Urt�me des Krieges die Nachtelfen in der Kunst des Kampfes aus.;;;;;;;;;;;x
+creature_ancient_war_desc;One of the toughest types of ancient, Ancients of War train night elves in the arts of weaponry.;;Als eine der z�hesten Arten der Urtume bilden Urtume des Krieges die Nachtelfen in der Kunst des Kampfes aus.;;;;;;;;;;;x
 creature_ancient_wind;Ancient of Wind;;Urtum des Windes;;;;;;;;;;;x
-creature_ancient_wind_desc;Ancients of Wind are able to tap into the more feral side of nature. Ancients of wind avoid combat but like other ancients, when enemies threaten nature, they battle with their allies.;;Die Urt�me des Windes sind in der Lage, die ungez�hmte Seite der Natur zu nutzen. Urt�me des Windes vermeiden den Kampf, aber wie andere Urt�me k�mpfen sie mit ihren Verb�ndeten, wenn Feinde die Natur bedrohen.;;;;;;;;;;;x
+creature_ancient_wind_desc;Ancients of Wind are able to tap into the more feral side of nature. Ancients of wind avoid combat but like other ancients, when enemies threaten nature, they battle with their allies.;;Die Urtume des Windes sind in der Lage, die ungez�hmte Seite der Natur zu nutzen. Urtume des Windes vermeiden den Kampf, aber wie andere Urtume k�mpfen sie mit ihren Verb�ndeten, wenn Feinde die Natur bedrohen.;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
 EVTDESC_WCRAC.300;You were walking in the forest when something fell from one of the branches. You lean to pick it up. In your hand you hold a little seed.;;Ihr seid im Wald spazieren gegangen, als etwas von einem der �ste fiel. Ihr beugt Euch vor, um es aufzuheben. In Eurer Hand haltet Ihr einen kleinen Samen.;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCRAC.300;You were walking in the forest when something fell from one of the branches. You lean over to pick it up. In your hand you hold a tiny seed.;;Ihr seid im Wald spazieren gegangen, als etwas von einem der �ste fiel. Ihr beugt Euch vor, um es aufzuheben. In Eurer Hand haltet Ihr einen kleinen Samen.;;;;;;;;;;;x
```

Reply:
Agreed on the first part, but I think the seed of an Ancient doesn't have to be tiny. Given their height once they are grown, I think "little" is small enough :)